### ISSUE TITLE
Is this fix the right thing to do?  fix for issue #75: making sure null columns gets a proper default value

### ISSUE THREAD

**Issue by carlcherry** [fluentcassandra/fluentcassandra#129]
Think about this scenario:
1. Add a new boolean column to an existing CF where you already have thousands or records
2. Let's assume you want the default to be True
3. Now the Fluent API returns the default of false for all records where the field is missing

So I cannot differentiate between a null field and one explicitly set to false (Or is there some way around this?)

**Comment by nberardi** [comment]
The alternative is to use a 3 state value.  `bool?` which will allow you to set to `true` or `false` or check for `null`.

**Comment by carlcherry** [comment]
This should be true for all primitive types - correct?

-----Original Message-----

**Comment by nberardi** [comment]
Yes, but we should probably have unit tests covering this, because I don't know if I have formally wrote one for nullable types.

**Comment by carlcherry** [comment]
I don't know the code very well.  From what I am seeing the idea would be to change the _value property in BooleanType to bool? And then fix any methods accordingly.  Is that where I should start?  The problem would be all the current clients using this library that don't use bool? - I assume they would all be broken.

-----Original Message-----

**Comment by nberardi** [comment]
I don't think you would change the BooleanType.  In which way are you using this, are you creating schema in CQL or using the record ad-hoc.

**Comment by carlcherry** [comment]
CQL3

-----Original Message-----
